lib/db_tests.js

```diff
@@ -10,6 +12,28 @@ var user2 = JSON.parse(dummyjson.parse(userTemplate));
 var user3 = JSON.parse(dummyjson.parse(userTemplate));
 var user1Guid = null;
 
+function dbListPromisified(type, restrictions) {
+  return new Promise(function(resolve, reject){
+    var params = {
+      "act": "list",
+      "type": type
+    };
+
+    if (restrictions && typeof restrictions === "function") {
```

Comment:
Is this condition really necessary? I would rather just do:
```
if (restrictions && typeof restrictions !== "function") {
    params = _.extend(params, restrictions);
}
```

Reply:
I was thinking about this... On one hand there could be just something like
`
if (restrictions) {
    params = _.extend(params, restrictions);
}
`
because I don't see any way, how can anything else get into the variable, but on the other hand I thought, if Peter or David put this condition into their cloud app, maybe there was a reason for that 😅